bin/bootstrap-cloud

```diff
@@ -122,6 +122,18 @@ if [[ ${__cloud} == aws ]]; then
     *) echo terraform apply result ${__ret} && exit ${__ret} ;;
   esac
 
+  if test -e ${__root}/config.yaml; then
+    __name=$(grep ^cluster_name: ${__root}/config.yaml | awk '{print $2}')
+  else
+    __name=dmtr-cluster
+  fi
+  if [[ $(kubectl config get-contexts | awk '{print $2}' | grep aws-${__name}) -eq 0 ]]; then
```

Comment:
This seems to be missing a `| wc-l` or similar

Reply:
Updated to use grep -o like other checks.